core/domain/stats_services.py

```diff
@@ -480,6 +480,11 @@ def get_visualizations_info(exp_id, state_name, interaction_id):
         if calc_output_domain_object is None:
             continue
 
+        # Don't show top unresolved answers calculation ouutput in stats of
+        # exploration.
+        if calculation_id == 'TopNUnresolvedAnswersByFrequency':
```

Comment:
Technically, better to put this on line 473. Then you don't even need to make the datastore GET request.

Note also that "output" is misspelled in the comment above.

Reply:
Hi @prasanna08 -- apart from the comment above, please note that backend tests are failing.

Reply:
Done.